feat: Tweak coding standards page IO-352 (#1579)
codacy/docs: docs/organizations/using-a-coding-standard.md

When you customize the tools or code patterns of a repository that follows the coding standard, Codacy warns you that the repository will stop following the coding standard and asks for your confirmation.

!!! important

>     [Tool configuration files](../repositories-configure/configuring-code-patterns.md#using-your-own-tool-configuration-files) override coding standards.

Author: Is this the gist? The original gives some more details, but I found it hard to follow. Let me know if I lost something!

Reviewer: Your sentence is correct but omits the detail that the coding standard will still affect the on/off status of the tools using configuration files.

I'm also concerned that it loses a bit of context that may be important for users reading this page because in your version the information is stated "from the point of view" of tool configuration files, which is a big jump away from the discussion around the concept of the coding standard.

My suggestion is that you clarify the information by splitting it into separate sentences:

- The coding standard turns tools with configuration files on and off
- However, the configuration files take precedence over the code patterns defined on the coding standard

Author: Keeping it singular for now:
```suggestion
    The coding standard turns tools with configuration files on and off. Tool configuration files, however, take precedence over the code patterns defined on the coding standard.
```

>     The coding standard turns tools with configuration files on and off. Tool configuration files, however, take precedence over the code patterns defined on the coding standard.

Reviewer 2: The original sentence is definitely very complex! I like your simplified version, I only feel that we need to reinforce that we are referring to the configuration files of the previously mentioned tools.

```suggestion
    The coding standard turns tools with configuration files on and off. Those tool configuration files, however, take precedence over the code patterns defined on the coding standard.
```

Reviewer: I agree! :+1:

1.  Name your coding standard and click **Create standard**.

    Optionally, select a repository for Codacy to use as a baseline when bootstrapping the tool and pattern configurations for the new coding standard. This is useful if you already have a configured repository that you wish to use as a template.

    ![Creating a new coding standard](images/coding-standard-create.png)

1.  Select the programming languages that the new coding standard should cover and click **Next: Tools and patterns**.

    The coding standard will only include configurations for the tools that support at least one of the selected languages.

    ![Selecting the languages for the coding standard](images/coding-standard-select-languages.png)

1.  Configure the tools and patterns of the coding standard and click **Next: Select and apply to repositories**.

    -   Toggle the tools to run when analyzing your code

>     -   For each enabled tool, configure the code patterns to use

Author: Again, it's a bit more concise like this—unless we want to explicitly mention Codacy

## Setting a coding standard as default {: id="set-default"}

> New repositories in your organization automatically follow the default coding standard.

Reviewer: There's an opportunity here to clarify what currently happens when the single coding standard isn't defined as "default" - which is that new repositories will use the "Codacy defaults".

This could then change depending on future product changes.

Author: ```suggestion
New repositories in your organization automatically follow the default coding standard or Codacy defaults if no default coding standard is set.
```
